index.js

```diff
@@ -2,81 +2,76 @@ module.exports = {
   env: {
     es6: true,
     node: true,
-    mocha: true
+    mocha: true,
   },
   plugins: ['mocha', 'prettier'],
-  extends: ['eslint-config-airbnb', 'prettier'],
+  extends: ['airbnb', 'prettier'],
   rules: {
-    // http://eslint.org/docs/rules/no-use-before-define
-    'no-use-before-define': [
-      2,
-      {
-        functions: false
-      }
-    ],
-
     // http://eslint.org/docs/rules/quotes
     quotes: [
       2,
       'single',
       {
         avoidEscape: true,
-        allowTemplateLiterals: true
-      }
+        allowTemplateLiterals: true,
+      },
     ],
 
     // http://eslint.org/docs/rules/comma-dangle
-    'comma-dangle': [2, 'never'],
-
-    // http://eslint.org/docs/rules/no-shadow
-    'no-shadow': [0],
```

Comment:
where did this one go?

Reply:
simon made the valid comment that we don't want to disable this. shadowed var re-use might cause issues. I agreed, so we took that disabling rule out, much like the other comments on safety.

Reply:
ah, good thought, thank you!